### PR TITLE
Submit task 1 of Emmadi Divya Srujana

### PR DESCRIPTION
**Pull request by EmmadiDivyaSrujana**

### CODE REVIEW

**Comment by MadhavBahlMD** [comment]
Please change the name of the folder from `Task 1` to `task1`

File: Emmadi Divya Srujana/task1/index.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        <label>Enter your username: </label>
+        <input type="text" name="Enter here" size=10><br><br>
+        <label>Enter your Email id</label>
+        <input type="email" name="Enter here" size=10><br><br>
+        <label>Enter your registeration number</label>
+        <input type="text" name="Enter here" size=10><br><br>
+        <label>Gender</label><br>
+        <input type="radio" name="Male">Male<br>
+        <input type="radio" name="Female">Female<br>
+        <input type="radio" name="Other">Other<br>
```
Comment (HarshKapadia2) [inline Emmadi Divya Srujana/task1/index.html:23]: Only one radio button should be active, @EmmadiDivyaSrujana .
For that, keep the `name` attribute the same in all three `input` tags and add the 'male', 'female' and 'others' to a `value` attribute.

Comment (EmmadiDivyaSrujana) [reply Emmadi Divya Srujana/task1/index.html]: Thank you for correcting me. I have resolved the issue.

**Comment by HarshKapadia2** [comment]
Thank you for making the necessary changes!
Keep contributing!

**Pull request merged**